src/workloads/scale/UpdateMillionDocsInTxn.yml

```diff
@@ -0,0 +1,78 @@
+# This workload is developed to test the amount of time it takes to update a million documents in
+# a single replica set transaction. At the moment, the average time taken raises above the default
+# limit, so until we add a way to manually increase a transaction's lifetime, we must raise the
+# lifetime of all transactions.
+
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/sharding"
+
+Actors:
+- Name: Loader
+  Type: Loader
+  Threads: 1
+  Phases:
+  - ExternalPhaseConfig:
+      Path: &PhasePath ../../phases/scale/DesignDocWorkloadPhases.yml
+      Key: Loader
+      Parameters:
+        CollectionCount: 1
+        DocumentCount: 1e6
+        BatchSize: 100000
+    Repeat: 1
+    Threads: 1
+  - &Nop {Nop: true}
+  - *Nop
+
+- Name: RaiseTransactionLifetimeLimit
```

Comment:
Blake, is this likely to go away in the future based on Server changes? If so, I would consider taking this step out and putting it as a pre-set in DSI in the test config part. (pre task or pre test, or just mongodb_setup). I think that will be easier to control. It also feels like that is about the system under test, as opposed to the actual test you are running. We could evens script that to do something different based on version numbers. 

That said, if this is always to be needed, then no harm having it here.

Reply:
The current view is that re offline conversation about this ticket

https://jira.mongodb.org/browse/SERVER-42976

that we may not actually do it, so I'd prefer leaving this here, because it's specific to the transaction we're trying to run, and if we reach a transaction timeout in other tests, that should count as a failure.

Reply:
That sounds reasonable. Do we need to make sure to unset the parameter afterwards? Will it survive a reboot if we run another test afterwards (like canaries).

Reply:
We set the parameter back to the default value at the end of this task, in the third phase. And no, this would not survive a reboot. It's sourced from a config file or command line parameter on startup.